Automattic/jetpack
# Fix VideoPress uploader editor pause text wrapping

Fixes Automattic/greenhouse#1355
Fixes Automattic/greenhouse#1339

#### Changes proposed in this Pull Request:

This PR fixes the text wrapping of the pause/resume text on the uploader editor.
It also fixes the custom poster link color when using Videomaker.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* On a Jetpack site with VideoPress
* Apply patch
* Download, install and activate Videomaker https://wordpress.org/themes/videomaker/
* `jetpack build plugins/jetpack`
* Create a new post
* Add a /columns component with 1 column
* Add a /video start a video upload and pause the upload
* ✅ The custom poster link should be visible
* :white_check_mark: The editor should look fine
* Click on the columns component and set the columns count to 2
* :white_check_mark: The pause/resume text should still be on one line, the flexbox might be wrapped
* Click on the columns component and set the columns count to 3
* :white_check_mark: It's probably ugly, but the pause/resume text should still be on one line, the flexbox might be wrapped
